nulib/dc-api-v2
# Major documentation overhaul: new layouts, pages, content, etc.

## Steps to test

1. Follow the instructions in the README to start the docs server locally
2. Docs should look like this:
![SCR-20230131-jic](https://user-images.githubusercontent.com/1395707/215870072-80442ad5-959a-4b58-a172-a758985dafe7.png)
3. Try out some of the features like the code examples.
